static/templates/presence_row.hbs

```diff
@@ -5,8 +5,18 @@
           href="{{href}}"
           data-user-id="{{user_id}}"
           data-name="{{name}}">
-            <span class="user-name">{{name}}</span>
-            {{> status_emoji status_emoji_info}}
+            {{#if right_sidebar_style.COMPACT}}
+                <span class="user-name">{{name}}</span>
+                {{> status_emoji status_emoji_info}}
+            {{else if right_sidebar_style.WITH_STATUS}}
```

Comment:
Let's write these conditionals as first "if right_sidebar_style.WITH_STATUS` and then a bare `else` clause resulting in the compact style. This will make our life easier for testing the "with avatar" option in the future -- as you've currently implemented it, if one has a development environment left on the "with avatar" option but not on the branch that adds its UI, the right sidebar will be broken, whereas if we do it as I suggest, it'll just fall back to the compact style default.